medusa/scene_exceptions.py

```diff
@@ -28,13 +28,14 @@
 from six import iteritems, text_type
 from . import app, db, helpers, logger
 from .indexers.indexer_config import INDEXER_TVDBV2
+from .session.core import MedusaSession
 
 exceptionsCache = {}
 exceptionsSeasonCache = {}
 
 exceptionLock = threading.Lock()
 
-xem_session = helpers.make_session()
+xem_session = MedusaSession()
```

Comment:
Can we disable this check in codacy? I tried disabling it. But I only have the option to disable all naming convention checks.